lib/Document/Adapter/Ghostscript.php

```
@@ -210,14 +210,18 @@ public function saveImage(string $imageTargetPath, $page = 1, $resolution = 200)
     /**
      * {@inheritdoc}
      */
-    public function getText(?int $page = null, ?Asset\Document $asset = null)
+    public function getText(?int $page = null, ?Asset\Document $asset = null, ?string $path = null)
     {
         try {
             if (!$asset && $this->asset) {
                 $asset = $this->asset;
             }
 
-            $path = $asset->getLocalFile();
+            if(!$path || !file_exists($path)) {
+                if(self::isFileTypeSupported($asset->getFilename())) {
+                    $path = $asset->getLocalFile();
+                }
+            }
```

@Corepex I guess we should throw an exception here in case `$path` is still null.

---

done https://github.com/pimcore/pimcore/pull/14630/commits/9b91edc67c6d2a7166acca2d862a88318bda9c9b (bzw. https://github.com/pimcore/pimcore/pull/14630/commits/9fe406802dce6ce44c9b83b2941fd2c5b8b06a14)